code/__HELPERS/unsorted.dm

```diff
@@ -1493,7 +1493,8 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 		/obj/item/reagent_containers/food/snacks/grown/mushroom,
 		/obj/item/reagent_containers/food/snacks/grown/nettle, // base type
 		/obj/item/reagent_containers/food/snacks/deepfryholder,
-		/obj/item/reagent_containers/food/snacks/clothing
+		/obj/item/reagent_containers/food/snacks/clothing,
+		/obj/item/reagent_containers/food/snacks //base type
```

Comment:
this shouldn't be needed when using `subtypesof`

Reply:
/obj/item/reagent_containers/food/snacks/grown/shell would cause this

Reply:
/obj/item/reagent_containers/food/snacks/store/bread as well

Reply:
should make a note in the food file that if you add something with a dummy parent you should update this proc

Reply:
Will do.